home/templates/partials/dashboard-base.html

```diff
@@ -73,7 +73,7 @@
         <div class="app-header-left">
           <div class="search-wrapper">
             <div class="input-holder">
-              <input type="text" class="search-input" placeholder="Type to
+              <input type="text" class="'show_salon" placeholder="Type to
```

Comment:
Why did you rename the search input class to show_salon 

Thought show_salon was a view name, could the rename have been a mistake?

Reply:
oh yeah this is mistake